pkg/kubernetes/api/core/v1/persistentvolumeclaim/build.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolumeclaim
```

Comment:
This package is not being used.

Reply:
We may need this in the upcoming PRs to get some additional attributes. So keep this for now.

Reply:
Also, in the upcoming PRs, it will be good to move away from these local to a common-kubernetes client, once we have hit GA with this and a few other repos.